packages/app-content-pages/src/screens/About/components/Highlights.js

```diff
@@ -0,0 +1,91 @@
+import { Anchor, Box, Button, Heading, Image, Paragraph, Text, Tip } from 'grommet'
+import { Trans, useTranslation } from 'next-i18next'
+import { CircleInformation } from 'grommet-icons'
+import styled from 'styled-components'
+
+const Relative = styled(Box)`
+  // This is necessary so the Contact Us section's header margin doesn't overlap the interactive icon in this section
+  position: relative;
+`
+
+export default function Highlights() {
+  const { t } = useTranslation()
+
+  return (
+    <>
+      <Heading
+        level={3}
+        size='1.5rem'
+        alignSelf='center'
+        weight='normal'
+        margin='0'
+        textAlign='center'
+      >
+        {t('AboutPage.highlights.subheading')}
+      </Heading>
+      <Paragraph>
+        <Trans
+          i18nKey={'AboutPage.highlights.paragraphs.one'}
+          t={t}
+          components={[
+            <Anchor
+              key='publications-page'
+              href='/publications' // after switch to app-root, this will need to be /about/publications
+            />
+          ]}
+        />
+      </Paragraph>
+      <Paragraph margin='0'>
+        {t('AboutPage.highlights.paragraphs.two')}
+      </Paragraph>
+      <Box
+        justify='center'
+        gap='xsmall'
+        margin={{ top: '30px' }}
+        direction='row'
+      >
+        <Image
+          alt=''
```

Comment:
Currently in PFE it looks like we have alt text for the Highlights book images along the lines of "Zooniverse Highlights Book Cover 2022." Is that something we could add for these images as well?

Reply:
Yes! I wasn't sure if we wanted a screen reader to read decorative images like these, but I added alt translation strings.